lib/index.js

```diff
@@ -210,12 +222,24 @@ api.set = async ({
   } else {
     query['account.email'] = email;
   }
-  const result = await database.collections.account.updateMany(
-    query, {
-      $set: {
-        [`meta.bedrock-authn-token.tokens.${type}`]: token
-      }
-    }, database.writeOptions);
+
+  let result;
+  if(type === 'nonce') {
+    result = await database.collections.account.updateMany(
+      query, {
+        $push: {
+          [`meta.bedrock-authn-token.tokens.${type}`]: token
+        }
```

Comment:
This seems like an ok approach to solving the problem. However, do we have sufficient test coverage to ensure it doesn't break anything else? Is this a backwards compatible change? cc: @mattcollier